src/display/editor/ink.js

```diff
@@ -581,6 +682,10 @@ class InkEditor extends AnnotationEditor {
    * @param {PointerEvent} event
    */
   canvasPointerleave(event) {
+    if (event.relatedTarget === null) {
+      // It can happen if the right-click menu is opened.
+      return;
+    }
```

Comment:
I really think that we should just *stop drawing* in this case, e.g. if the context-menu is opened, since having drawing continue if the mouse if moved outside of the context-menu seems unexpected/wrong.